File: rust/dreive/src/customer.rs
pub struct customer {
    pub name: String,
    pub id: u32,
    pub age:u8,
}
impl customer {
    pub fn new(name: String, id: u32, age:u8) -> Self {
        customer { name, id, age }
    }

    pub fn print_customer(&self) {
        println!("Customer Name: {}, ID: {}, Age: {}", self.name, self.id, self.age);
    }
}

File: python/project/routes/customer.py
from flask import Blueprint
from controllers.customer import get_customers, create_customer, update_customer, delete_customer

customer_bp = Blueprint('customer', __name__)

customer_bp.route('/customers', methods=['GET'])(get_customers)
customer_bp.route('/customers', methods=['POST'])(create_customer)
customer_bp.route('/customers/<int:id>', methods=['PUT'])(update_customer)
customer_bp.route('/customers/<int:id>', methods=['DELETE'])(delete_customer)
